### DISCUSSION TITLE
opencanary logs parser for little visual interface

### DISCUSSION THREAD

**Discussion by 7h30th3r0n3** [thinkst/opencanary#312] [Show and tell]
Hey ! 
Check this out : 
https://github.com/7h30th3r0n3/OpenCanaryChecker

**Comment by 1Stronk** [comment]
sick

**Comment by SecuriLee** [comment]
that would be cool to pump to a php or html page :)

**Reply by SecuriLee** [reply]
My 2353489841 byte log file kills this script :D 
You might want to consider the following:
- variable for the log file location (I had to change it in two locations)
- if the log file is beyond a certain length, limit what you are looking at for performance reasons or tell the user

I feed my data into Splunk but this is something cute that could feed my websites

**Reply by 7h30th3r0n3** [reply]
Yeah haha ! It's a simple script that probably need optimisation, I'm using it on a one month log file and flush the log after grabbing interesting things, so yeah a 2Go log file is probably too mutch to handle properly

**Comment by jayjb** [comment]
Hi @7h30th3r0n3,

This is so cool. Im sorry for the delayed response (and thanks so much for sharing this).

**Reply by 7h30th3r0n3** [reply]
No worries, it is there for those who need it. Thx is a small fast made script and it is mainly to easily show what it recovers when starting up or at the start of the harvest, because big data seem to crash the thing haha but it was quite useful to me to see where it was rather than doing that in the logs directly
